test/e2e/specs/blocks/blocks__smoke-test-across-gb-edge-versions.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @group gutenberg
+ *
+ * This spec is meant to be a lightweight test of popular/relevant blocks across
+ * Gutenberg versions. This will run in a site stickered with `gutenberg-edge` and
+ * will load a specific post with pre-configured blocks in it. It will then verify
+ * that none of these blocks are error'ing or invalidating in the editor.
+ *
+ * This spec is not meant to replace specific block sepcs. First of all, specific
+ * block specs should ideally be added upstream, and not here in Calypso, unless
+ * the block is developed as part of the Calypso monorepo. This test is also not
+ * meant to test specific block behavior, but instead to verify if they continue
+ * working across GB versions, during the GB upgrade process in WPCOM.
+ *
+ * To avoid any confusion, the tests here will only run if the GUTENBERG_EDGE env
+ * var is set.
+ */
+import { GutenbergEditorPage, TestAccount, envVariables } from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+
+const test = envVariables.GUTENBERG_EDGE ? it : it.skip;
```

Comment:
I love this kind of syntax, but we actually already have a couple wrappers for this already [here](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/jest-conditionals.ts)!

Would you be up for using one of those two just for similarity among specs?